--- src/configure.orig	2025-03-05 14:23:14.797405543 +0100
+++ src/configure	2025-03-05 14:24:01.777995712 +0100
@@ -55425,13 +55425,15 @@ sysv4 | sysv4.3*)
     ;;
   esac
   ;;
-
 tpf*)
   lt_cv_deplibs_check_method=pass_all
   ;;
 os2*)
   lt_cv_deplibs_check_method=pass_all
   ;;
+genode*)
+  lt_cv_deplibs_check_method=pass_all
+  ;;
 esac
 
 fi
@@ -59977,6 +59979,11 @@ $as_echo "$lt_cv_irix_exported_symbol" >
       hardcode_shlibpath_var=no
       ;;
 
+    genode*)
+      archive_cmds='$CC $libobjs $deplibs '$LDLIBS_SHARED' -o $lib'
+      archive_expsym_cmds=''
+      ;;
+
     *)
       ld_shlibs=no
       ;;
@@ -61048,6 +61055,13 @@ uts4*)
   shlibpath_var=LD_LIBRARY_PATH
   ;;
 
+genode*)
+  dynamic_linker="Genode ld.lib.so"
+  shrext_cmds=.lib.so
+  libname_spec='$name'
+  library_names_spec='$libname$shared_ext'
+  ;;
+
 *)
   dynamic_linker=no
   ;;
@@ -63320,6 +63334,10 @@ fi
         ld_shlibs_CXX=no
         ;;
 
+      genode*)
+        ld_shlibs_CXX=yes
+        ;;
+
       *)
         # FIXME: insert proper C++ library support
         ld_shlibs_CXX=no
@@ -65013,6 +65031,13 @@ uts4*)
   shlibpath_var=LD_LIBRARY_PATH
   ;;
 
+genode*)
+  dynamic_linker="Genode ld.lib.so"
+  shrext_cmds=.lib.so
+  libname_spec='$name'
+  library_names_spec='$libname$shared_ext'
+  #dynamic_linker=yes
+  ;;
 *)
   dynamic_linker=no
   ;;
